keras/models.py

```diff
@@ -76,6 +88,49 @@ def standardize_weights(y, sample_weight=None, class_weight=None):
     else:
         return np.ones(y.shape[:-1] + (1,))
 
+def sequential_from_yaml(yamlString):
+    '''
+        Returns a compiled Sequential model generated from a local yaml file,
+        which is either created by hand or from Sequential.to_yaml
+    '''
+    modelYaml = yaml.load(yamlString)
+    model = Sequential()
+
+    class_mode = modelYaml.get('class_mode')
+    theano_mode = modelYaml.get('theano_mode')
+    loss = globals()[modelYaml.get('loss')]
```

Comment:
> Train a model and store every bit of it, including optimizers, in case you want to pick it up later for further training etc. In this case it makes sense to have the compiled version at hand. At least I would like to have the option to have a compiled version.

Sounds very useful, but there are further problems with implementing this: for instance, how do you save the state of the optimizer for further training? That state is contained in shared Theano variables.

I have no strong opinion either way, the main argument I see for skipping compilation is the use of custom objectives (custom optimizers are relatively unlikely, much like custom regularizers and constraints, however custom objectives are fairly common).